api/channel.go

```diff
@@ -1300,3 +1301,27 @@ func viewChannel(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	ReturnStatusOK(w)
 }
+
+func getChannelMembersByIds(c *Context, w http.ResponseWriter, r *http.Request) {
```

Comment:
I assume this is only going to be called once you open the Add Members modal?  If that's the case then it's OK, no need to optimize.